tests/robot-tests/tests/libs/common.robot

```diff
@@ -555,3 +556,20 @@ user checks page contains other release
 user checks page does not contain other release
     [Arguments]   ${other_release_title}
     user checks page does not contain element   xpath://li[@data-testid="other-release-item"]/a[text()="${other_release_title}"]
+
+user navigates to find statistics page 
```

Comment:
I'd rename this keyword `user navigates to find statistics page on public frontend`